step/checks.go

```diff
@@ -0,0 +1,47 @@
+package step
+
+import (
+	"fmt"
+	"os"
+)
+
+const unavailableAnnotation = `You have added the **Activate Bitrise Build Cache for Gradle** add-on step to your workflow.
+
+However, it has not been activated for this workspace yet. Please contact [support@bitrise.io](mailto:support@bitrise.io) to activate it.
+
+Build cache is not activated in this build.`
+
+const gradleHome = "~/.gradle"
+
+func (step RemoteCacheStep) ensureFeatureEnabled() (bool, error) {
+	isEnabled := step.envRepo.Get("BITRISEIO_BUILD_CACHE_ENABLED") == "true"
+
+	if !isEnabled {
+		step.logger.Warnf(unavailableAnnotation)
+
+		cmd := step.commandFactory.Create("bitrise", []string{
+			":annotations",
+			"annotate",
+			unavailableAnnotation,
+			"--style", "warning",
+		}, nil)
+		cmdOut, err := cmd.RunAndReturnTrimmedCombinedOutput()
+		if err != nil {
+			return false, fmt.Errorf("failed to create annotation: %s", cmdOut)
+		}
```

Comment:
I think a
```
return false, nil
```
if missing here after the err checking. 

Because if I am reading this correctly then when the `BITRISEIO_BUILD_CACHE_ENABLED` is missing then we print out the warning and execute the annotation. If the annotation command succeeded then we simply exit the if block and return a true value. Which indicated that the feature is enabled but it should not be.

Reply:
Indeed! Thank you, it's fixed now